src/Shared/CertificateGeneration/CertificateManager.cs

```diff
@@ -770,6 +772,18 @@ private static void RemoveCertificateFromUserStore(X509Certificate2 certificate)
         }
     }
 
+    protected virtual void RemoveCertificateFromUserStoreCore(X509Certificate2 certificate)
+    {
+        using var store = new X509Store(StoreName.My, StoreLocation.CurrentUser);
+        store.Open(OpenFlags.ReadWrite);
+        var matching = store.Certificates
+            .OfType<X509Certificate2>()
+            .Single(c => c.SerialNumber == certificate.SerialNumber);
+
+        store.Remove(matching);
+        store.Close();
```

Comment:
This is not necessary in this case because of the `using var store` on line 777, which will call `Dispose()` automatically which will in turn call `Close()`.